controllers/ControllerComment.php

```diff
@@ -47,30 +41,32 @@ public function addCommentPost(){
 			$good = $this->_commentManager->addComment($userId, $comment, $postId['id']);
 
 			if ($good){
-				header("Location: http://".$this->_server."/Post/".$postId['id']);
+				header("Location: http://".$this->_server."/Post/viewPost/".$postId['id']);
 			}else{
 				echo 'Une erreur c\'est produite veuillez recommencer !' ;
 			}
 		}
 	}
 
-	private function updateComment(){
+	public function updateComment(){
 
-		$key = (new Session)->getKey('post');
-		$commentId = $this->_method[3];
+		$keyPost = (new Session)->getKey('post');
+		$keyUser = (new Session)->getKey('user');
 		$post = $this->_params;
 
 		if (isset($post['message'])){
-			$this->checkComment($post['message'], $commentId, $key['id']);
+			$this->checkComment($post['message'], $this->_slug, $keyPost['id']);
 		}
 
 
 
-		$content = $this->_commentManager->oneComment((int) $commentId);
+		$content = $this->_commentManager->oneComment((int) $this->_slug);
 
 		echo $this->_twig->render( 'comment.twig', ['logged'=> TRUE,
 		                                            'server' => $this->_server,
 		                                            'id' => $content['id'],
+		                                            'admin' => $keyUser['admin'],
```

Comment:
Codacy found an issue: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$keyUser['admin']'.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6061935)